Assets/VRTK/Scripts/Presence/VRTK_BodyPhysics.cs

```diff
@@ -1035,13 +1038,15 @@ protected virtual void StartFall(GameObject targetFloor)
             isMoving = false;
             isLeaning = false;
             onGround = false;
+            fallMinTime = Time.time + (Time.fixedDeltaTime * 3.0f); // Wait at least 3 fixed update frames before declaring falling finished 
```

Comment:
I think whenever you mess with the collider it wakes up. We are constantly changing it's shape and position so I don't think that works. I played with it last year and never came to a good solution.